app/Main.hs

```diff
@@ -1,13 +1,25 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
 module Main where
 
+import Data.Aeson
```

Comment:
This is providing: `FromJSON`, `withObject`, and `(.:)`
I usually don't specify what I'm importing from Aeson because I end up using so many methods and operators from it that it gets tedious quickly. For instance

```haskell
import Data.Aeson (FromJSON, ToJSON, withObject, (.:), (.:?), (.:!), (.!=), json, withArray, withBool, withText)
```

It might make sense to import specific methods, I just haven't historically.